pkg/downwardmetrics/downwardmetrics.go

```diff
@@ -17,6 +19,8 @@ const (
 	DownwardMetricsChannelSocket    = DownwardMetricsChannelDir + "/downwardmetrics.sock"
 )
 
+var DownwardMetricsNotEnabledError = errors.New("DownwardMetrics feature is not enabled")
```

Comment:
```suggestion
var NotEnabledError = errors.New("DownwardMetrics feature is not enabled")
```

When used as `downwardmetrics.NotEnabledError` this avoids repetition.